scripts/test-infra/verify.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+# Install deps
+gobinpath="$(go env GOPATH)/bin"
+curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b "$gobinpath" v1.42.1
+export PATH=$PATH:$(go env GOPATH)/bin
+
+# Run verify steps
+make gofmt-verify
```

Comment:
I thought this didn't produce changes, just fail.

Should we add `set -e` at the top?

Reply:
`#!/bin/bash -e` is at the header

Reply:
Why not use `#!/usr/bin/env bash`?